openmetadata-spec/src/main/resources/json/schema/api/data/createQuery.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$id": "https://open-metadata.org/schema/api/data/createQuery.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "CreateQueryRequest",
+  "description": "Create Query Request",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.api.data.CreateQuery",
+  "javaInterfaces": [
+    "org.openmetadata.schema.CreateEntity"
+  ],
+  "properties": {
+    "name": {
+      "description": "Name of a entity to which the query belong",
+      "$ref": "../../type/basic.json#/definitions/entityName"
+    },
+    "displayName": {
+      "description": "Display Name that identifies this query.",
+      "type": "string"
+    },
+    "description": {
+      "description": "Description of the query instance.",
+      "$ref": "../../type/basic.json#/definitions/markdown"
+    },
+    "owner": {
+      "description": "Owner of this entity",
+      "$ref": "../../type/entityReference.json",
+      "default": null
+    },
+    "tags": {
+      "description": "Tags for this table",
+      "type": "array",
+      "items": {
+        "$ref": "../../type/tagLabel.json"
+      },
+      "default": null
+    },
+    "query": {
+      "description": "SQL Query definition.",
+      "type": "string"
+    },
+    "duration": {
+      "description": "How long did the query took to run in seconds.",
+      "type": "number"
+    },
+    "users": {
+      "description": "List of users who ran this query.",
+      "type": "array",
+      "items": {
+        "$ref": "../../type/entityReference.json"
+      },
```

Comment:
Instead of entityReference, we can just turn this into the names of users. We stopped using EntityReference in create requests.

Reply:
we can have teams as well creating queries